helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -285,106 +332,87 @@ func TraerDetalle(id int) (Solicitud map[string]interface{}, outputError map[str
-								"FechaVistoBueno":   data_["FechaVistoBueno"],
-								"Estado":            data["FechaCreacion"],
-								"Activo":            data["FechaCreacion"],
-								"Elementos":         Elementos__,
-							}
-							return Solicitud, nil
-
-						} else {
-							logs.Error(err)
-							outputError = map[string]interface{}{
-								"funcion": "/TraerDetalle",
-								"err":     err,
-								"status":  "500",
-							}
-							return nil, outputError
-						}
-					} else {
-						return nil, err
-					}
-				} else {
-					return nil, err
-				}
-			} else {
-				logs.Error(err)
-				outputError = map[string]interface{}{
-					"funcion": "/TraerDetalle",
-					"err":     err,
-					"status":  "502",
-				}
-				return nil, outputError
-			}
-		} else {
-			logs.Error(err)
-			outputError = map[string]interface{}{
-				"funcion": "/TraerDetalle",
-				"err":     err,
-				"status":  "500",
-			}
-			return nil, outputError
-		}
+	var (
+		movimiento *models.Movimiento
+		detalle    models.FormatoBaja
+	)
+	Baja = new(models.TrBaja)
+
+	// Se consulta el movimiento
+	if movimientoA, err := movimientosArkaHelper.GetMovimientoById(id); err != nil {
+		return nil, err
 	} else {
+		movimiento = movimientoA
+	}
+
+	if err := json.Unmarshal([]byte(movimiento.Detalle), &detalle); err != nil {
 		logs.Error(err)
 		outputError = map[string]interface{}{
-			"funcion": "/TraerDetalle",
+			"funcion": "TraerDetalle - json.Unmarshal([]byte(movimiento.Detalle), &detalle)",
 			"err":     err,
 			"status":  "502",
 		}
 		return nil, outputError
 	}
+
+	// Se consulta el detalle del funcionario solicitante
+	if detalle.Funcionario > 0 {
+		if funcionario, err := tercerosMidHelper.GetInfoTerceroById(detalle.Funcionario); err != nil {
+			return nil, err
+		} else {
+			Baja.Funcionario = funcionario
+		}
+	}
+
+	// Se consulta el detalle del revisor si lo hay
+	if detalle.Revisor > 0 {
+		if revisor, err := tercerosMidHelper.GetInfoTerceroById(detalle.Revisor); err != nil {
+			return nil, err
+		} else {
+			Baja.Revisor = revisor
+		}
+	}
+
+	// Se consulta el detalle de los elementos relacionados en la solicitud
+	if len(detalle.Elementos) > 0 {
+		if elementos, err := GetDetalleElementos(detalle.Elementos); err != nil {
+			return nil, err
+		} else {
+			Baja.Elementos = elementos
+		}
+	}
+
+	// Se consulta el detalle de los elementos relacionados en la solicitud
+	query := "query=MovimientoId__Id:" + strconv.Itoa(id)
+	if soportes, err := movimientosArkaHelper.GetAllSoporteMovimiento(query); err != nil {
+		return nil, err
+	} else if len(soportes) > 0 {
+		Baja.Soporte = soportes[0].DocumentoId
+	}
+
+	Baja.Id = movimiento.Id
+	Baja.TipoBaja = movimiento.FormatoTipoMovimientoId
+	Baja.Consecutivo = detalle.Consecutivo
+	Baja.Observaciones = movimiento.Observacion
+
+	return Baja, nil
+
 }
 
 func GetDetalleElemento(id int) (Elemento *models.DetalleElementoBaja, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetDetalleElemento hace tal cosa"
```suggestion
// GetDetalleElemento ...
func GetDetalleElemento(id int) (Elemento *models.DetalleElementoBaja, outputError map[string]interface{}) {
```